chore(curiculum): adding video ids for audio and video lecture block (#58532)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/lecture-working-with-audio-and-video/6733d829d983c008d2db41a1.md

id: 6733d829d983c008d2db41a1

> title: What Is Codecs and How Does It Work?

Reviewer: ```suggestion
title: What Are Codecs and How Do They Work?
```

challengeType: 19
videoId: Tr7ruMKuEIs

> dashedName: what-is-codecs-and-how-does-it-work

Reviewer: ```suggestion
dashedName: what-are-codecs-and-how-do-they-work
```

Reviewer: Oh okay :+1:

---

# --description--

Watch the video and answer the questions below.

# --transcript--

> What Is Codecs and How Does It Work?

Reviewer: ```suggestion
What are codecs and how do they work?
```
I guess we should change the title as well
